styles/utils.css

```diff
@@ -7,7 +7,10 @@
     --purple: purple;
     --skyblue: skyblue;
     --red: red;
+    --light-coral: lightcoral;
     --green: green;
+    --light-green: lightgreen;
+    --yellow: yellow;
     --pink: pink;
     --primary-button-hover: rgb(21, 50, 180);
     --secondary-button-hover: rgb(236, 233, 233);
```

Comment:
Use the same color code everywhere.

Reply:
It's intentional because of different use cases.